### PR TITLE
Add reducers

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It's my Bookstore project and I added the following features:

- Configured the Redux store.
- Added the booksSlice and the categoriesSlice.
- Added the checkStatus reducer.
- Added the addBook and removeBook reducers.

### CODE REVIEW

File: src/features/store.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import categoriesReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    categories: categoriesReducer,
+  },
+});
+
+export default store;
```
Comment (Shedrack-Sunday) [inline src/features/store.js]: - [ ] Good job on your work 👏 . However, kindly follow the `ducks pattern` [Link](https://github.com/erikras/ducks-modular-redux) for your Redux files and all file folders As instructed in the project requirements 👍 

### Expected:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @ivangonzalez224         👋 

# CHANGES REQUIRED ♻️

- Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the React features ✔️
- Good use of Redux ✔️  
    
# Required Changes  🌵  

- [ ] You've done well, However, kindly check some key point note I pointed out to meet the requirements before moving to the next project 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/features/store.js (hunk 1 above)
Comment (ivangonzalez224) [reply src/features/store.js]: Hi @Shedrack-Sunday ,
Thank you for the review, I don't quite understand the  ducks pattern very well, in this case do you mean that I just have to change the name of the folder called 'features' to 'redux'? or is there anything else to change ?

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ivangonzalez224, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

- Descriptive Pull Request. :heavy_check_mark:
- Action and Reducer functionality. :heavy_check_mark:
- Ducks pattern file structure. :heavy_check_mark:
- `@reduxjs/toolkit` & `react-redux` present in package.json :heavy_check_mark:
- Linters environment properly setup :heavy_check_mark:
- Good use of Git flow :heavy_check_mark:

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**